transformers/storage/val/keys_loader.go

```diff
@@ -0,0 +1,59 @@
+// VulcanizeDB
+// Copyright © 2018 Vulcanize
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package val
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	mcdStorage "github.com/makerdao/vdb-mcd-transformers/transformers/storage"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/storage"
+	vdbStorage "github.com/makerdao/vulcanizedb/libraries/shared/storage"
+	"github.com/makerdao/vulcanizedb/libraries/shared/storage/types"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+)
+
+const (
+	Has = "has"
+	Val = "val"
+)
+
+var (
+	HasKey      = common.HexToHash(vdbStorage.IndexZero)
+	HasMetadata = types.GetValueMetadata(Has, nil, types.Uint256)
```

Comment:
I suppose I should add types.bool here? Just felt strange that it was the first one.